src/Classes/Controllers/AddContactController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+
+namespace Portal\Controllers;
+
+
+class AddContactController
+{
+
+    private $hiringPartnerModel;
+
+    /**
+     * AddContactController constructor.
+     * @param $hiringPartnerModel
+     */
+    public function __construct($hiringPartnerModel)
+    {
+        $this->hiringPartnerModel = $hiringPartnerModel;
+    }
+
+
```

Comment:
You need an invoke too - the method on the model you'll need to call is `addNewContact`. 
Look at `CreateHiringPartnerController` or `AddEventController` for examples.